src/main/java/org/broadinstitute/hellbender/tools/walkers/GenotypeGVCFs.java

```diff
@@ -221,6 +224,16 @@ protected GenomicsDBOptions getGenomicsDBOptions() {
     @Override
     public boolean useVariantAnnotations() { return true;}
 
+    @Override
+    public List<? extends CommandLinePluginDescriptor<?>> getPluginDescriptors() {
+        GATKReadFilterPluginDescriptor readFilterDescriptor = new GATKReadFilterPluginDescriptor(getDefaultReadFilters());
+        return useVariantAnnotations()?
+                Arrays.asList(readFilterDescriptor, new GATKAnnotationPluginDescriptor(
+                        new GenotypeGVCFsAnnotationArgumentCollection(),
```

Comment:
Also, make sure that when you add the instance variable, you don't annotate it as an `@ArgumentCollection` (and maybe even add a comment saying that `@ArgumentCollection` is left out on purpose) since if you do the parser will complain about duplicate arg names.